### DISCUSSION TITLE
Is package maintained?

### DISCUSSION THREAD

**Discussion by ortonomy** [ingram-projects/animxyz#102] [Q&A]
I just discovered this package, but it's not been touched for 6 months - I'd love to use it in my vue project as vue gets very little love from Animation lib creators. 

But... I don't see any updates? Is the creator even on this any more?

**Comment by mattaningram** [comment]
Hi @ortonomy we are still going to work on AnimXYZ we have just been working on some new projects that have been taking most of @milesingrams and my time. 

Please let us know here or in issues of any bugs or feature requests you might have!
